src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -19,7 +20,7 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
+        String trimmedArgs = StringUtil.stripFlags(args.trim());
```

Comment:
put trim inside the stripFlags function?

Reply:
hmm I think the name `stripFlags` is restricted to removing the words that start with `-`, and one would not expect it to also trim the arguments. what do you think?